bindings/python/pymongocrypt/crypto.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2019-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Internal crypto callbacks for libmongocrypt."""
+
+import os
+import traceback
+
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.primitives.hashes import Hash, SHA256, SHA512
+from cryptography.hazmat.primitives.hmac import HMAC
+from cryptography.hazmat.backends import default_backend
+
+from pymongocrypt.binary import _to_bytes, _write_bytes
+from pymongocrypt.binding import ffi, lib
+from pymongocrypt.compat import str_to_bytes
+
+
+def _callback_error_handler(exception, exc_value, tb):
+    """Set the mongocrypt_status_t on error."""
+    # From cffi docs: "First check if traceback is not None (it is None e.g.
+    # if the whole function ran successfully but there was an error converting
+    # the value returned: this occurs after the call)."
+    if tb is None:
```

Comment:
I love that the cffi docs agree with me about "is not None". :-)

It's also weird that you return None here, but False later.

Reply:
Fixed.